serde/src/main/java/com/amazon/ionhiveserde/caseinsensitivedecorator/IonSequenceCaseInsensitiveDecorator.java

```diff
@@ -226,4 +226,9 @@ public void add(final IonValue ionValue) {
             listIterator.add(ionValue);
         }
     }
+
+    @Override
+    public String toString() {
+        return ionSequence.toString();
+    }
```

Comment:
Can this be moved to `IonContainerCaseInsensitiveDecorator` to avoid having to duplicate it in both `IonSequenceCaseInsensitiveDecorator` and `IonStructCaseInsensitiveDecorator`?

Also, when does this get used? Are containers not covered by `IonValueToStringObjectInspector`?

Reply:
Thanks. I thought we might have different behaviors for struct and sequence but `toString` method already handle them.